### DISCUSSION TITLE
I purchased the license, but encountered the following error while downloading. It can be downloaded normally before purchase

### DISCUSSION THREAD

**Discussion by foreverxhj** [aclap-dev/video-downloadhelper#517] [Help and Q&A]
### Your question

{
  "version": "9.0.1.8",
  "target": "google",
  "channel": "beta",
  "page_url": "https://live.vhall.com/v3/lives/embedclientfull/watch/381750098?k=eyJ3ZWJpbmFyX2lkIjoiMzgxNzUwMDk4IiwibWVldGluZ19pZCI6IjY0MjA4IiwibWVtYmVyX2lkIjoiMjE4MjUyMTMiLCJ0aW1lIjoxNzE2MTY4MTQxfQ%3D%3D&k_id=228249&state=64208&nickname=%E8%AE%B8&email=21825213-zhiqu%40scrmtech.com&k_only_watch=1",
  "error": {
    "details": "SideDownload error: : [tls @ 0x12be13650] IO error: Connection reset by peer\n[in#0 @ 0x12be11f80] Error opening input: Connection reset by peer\nError opening input file https://bdhlsvodhls02.vhallyun.com/vhallyun/vhallrecord/4f98e548/20240308143640_4f98e548/record.m3u8?token=1B507378_NjgyMTM5MTYwXzY2NTNFMDRGX05HWTVPR1UxTkRnX2RtbHphWFJmZGpFM09UQTFOemd6TnpNeE5qVTFNemt6TWpnX3ZvZA.\nError opening input files: Connection reset by peer\n",
    "downloadable_id": "downloadable_8626508519403426",
    "error": "coapp_failure",
    "id": "coapp_failure_e1666750-461a-4a12-8aac-6782bd5fb8ff"
  }
}

### Addon & CoApp version and details

I purchased the license, but encountered the following error while downloading. It can be downloaded normally before purchase

**Comment by paulrouget** [comment]
License or not, it should work the same way. Do a force reload of the page: press Chrome reload button while holding the shift key down.
